shared/db/collections/sentences-meta.js

```diff
@@ -121,6 +121,56 @@ export default class SentencesMeta {
     });
   }
 
+  async correctApprovals(locale) {
+    const collection = await this.getCollection(locale);
+    const records = await this.getAllPaginated(locale);
+    console.log(`Found ${records.length} records to analyze for ${locale}`);
+
+    const adjustedSentences = records.map((record) => {
+      const isApproved = this.checkIfApproved(record);
+      if (typeof isApproved === 'undefined' || isApproved === record.approved) {
+        // nothing to do here
+        return;
+      }
+
+      record.approved = isApproved;
+      record.approvalDate = Date.now();
+
+      return record;
+    }).filter(Boolean);
+
+    console.log(`Found ${adjustedSentences.length} to adjust`);
+
+    const results = await collection.batch(c => {
+      adjustedSentences.forEach(record => {
+        c.updateRecord(record, {
+          safe: true,
+          last_modified: record.last_modified,
+        });
+      });
+    });
+
+    const adjustedResults = [];
+    const errors = [];
+    results.forEach(result => {
+      if (result.status === 200 || result === 201) {
```

Comment:
That seems wonky^^ Especially since the number `201` typically has no body property.